src/components/Dashboard/Beneficiary/Beneficiary.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState, useEffect, useContext } from "react";
+import { areSameAddress } from "../../../modules/ethereum/utils";
+import { Grid } from "semantic-ui-react";
+import Icon from "../../../images/grant_icon.svg";
+import ButtonIcon from "../../../images/proposal_button_icon.svg";
+import { Header, Button } from "decentraland-ui";
+import { useIntl, FormattedMessage } from "react-intl";
+import useResponsive from "../../../hooks/useResponsive";
+import Responsive from "semantic-ui-react/dist/commonjs/addons/Responsive";
+import { DaoInitiativeContext } from "../../../context/DaoInitiativeContext";
+import { openInNewTab } from "../../../utils";
+
+import "./Beneficiary.css";
+
+function Beneficiary(props) {
+  const { address } = props;
+
+  const [proposals, setProposals] = useState(null);
+  const [daoProposal, setDaoProposal] = useState(false);
```

Comment:
`daoProposal` seems like a value storing a Proposal. Maybe `isDaoProposal`? Also does this need to be in a state? Can't it be in a simple boolean variable?